spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/spanner/GcpSpannerAutoConfiguration.java

```diff
@@ -64,6 +64,8 @@
 
 	static class CoreSpannerAutoConfiguration {
 
+		private final String host;
```

Comment:
The same problem here: an  emulator matter has to be separate from the standard configuration, even if a difference only with the one property.